src/scss/abstracts/_mixins.scss

```diff
@@ -150,11 +150,10 @@
 			}
 		}
 
-		// Home page cards: hide background images on UIO contrast themes
-		.blue,
-		.green,
-		.yellow {
+		// Home page cards: hide background images and adjust box shadows on UIO contrast themes
+		.card {
```

Comment:
Looks good @cindyli!

Add hover and focus styling for card box-shadows. Should look like the code chunk below.

```css
		&:hover,
		&:focus {
			box-shadow: rem(10) rem(10) rem(10) currentColor;
		}
```

Also `.card:focus` and `.card:hover` selectors should be removed on lines 114 and 115 respectively.